docs/section5/Rule5-37.md

```diff
@@ -0,0 +1,83 @@
+
+# Envelope - Rule 5-37  
+
+**Rule ID:** 5-37  
+**Rule Description:** Skylight U-factors for residential, non-residential and semi-heated spaces in the baseline model must match the appropriate requirements in Table G3.4-1 through G3.4-8.  
+**Rule Assertion:** B-RMR subsurface: U_factor = expected value  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(e) Building Envelope Modeling Requirements for the Baseline building
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**
+
+  1. get_surface_conditioning_category()
+  2. get_opaque_surface_type()
+  3. get_scc_skylight_roof_ratios()
+  4. data_lookup()
```

Comment:
similar to comment on 5-24, could we please rename get_scc_skylight ... to get_building_scc_skylight... to clarify that this is getting the SRR for the entire building and not for each building segmnt.

Reply:
Updated.